music_collection/music_collection/settings.py

```diff
@@ -63,7 +63,7 @@
 TEMPLATES = [
     {
         'BACKEND': 'django.template.backends.django.DjangoTemplates',
-        'DIRS': [],
+        'DIRS': [os.path.join(BASE_DIR, "templates")],
```

Comment:
(pathlib)[https://docs.python.org/3/library/pathlib.html] jest super, polecam się zapoznać z tą libką bo na pewno przyjdzie Wam jeszcze walczyć ze ścieżkami, tutaj np. można taki przykład uprościć do:
```suggestion
        'DIRS': BASE_DIR / "templates",
```